flake.nix

```diff
@@ -3,40 +3,39 @@
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
-    utils.url = "github:numtide/flake-utils";
     flake-compat = {
       url = "github:edolstra/flake-compat";
       flake = false;
     };
   };
 
-  outputs = { self, nixpkgs, nix, flake-utils, ... }@inputs:
+  outputs = { self, nixpkgs, nix, ... }@inputs:
+    let
+      forSystems = nixpkgs.lib.genAttrs [ "x86_64-linux" ];
+    in
     {
+      overlays.default = final: prev: {
+        vault-push-approles =
+          final.callPackage ./scripts/vault-push-approles.nix { };
+        vault-push-approle-envs =
+          final.callPackage ./scripts/vault-push-approle-envs.nix { };
+      };
 
-    overlay = final: prev: {
-      vault-push-approles =
-        final.callPackage ./scripts/vault-push-approles.nix { };
-      vault-push-approle-envs =
-        final.callPackage ./scripts/vault-push-approle-envs.nix { };
-    };
-
-    nixosModules.vault-secrets = import ./modules/vault-secrets.nix;
-    darwinModules.vault-secrets = import ./modules/vault-secrets-darwin.nix;
```

Comment:
Did you intentionally get rid of `darwinModules`?

Reply:
No that was probably a copy and paste error.